kakao/README.md

```diff
@@ -4,23 +4,22 @@ This sample shows how to Sign in Firebase using [Kakao Login](https://developers
 
 ## Setup the sample
 
-### Kakao's creation and setup
+### Kakao app creation and setup
 
  1. Follow the instruction in [Kakao Developers Doc](https://developers.kakao.com/docs/android#getting-started-create-app) to setup a Kakao developer account and your application.
```

Comment:
This should maybe reference both Android and iOS docs to be more platform-agnostic.